mevislab.github.io/content/tutorials/basicmechanisms/dataimport.md

```diff
@@ -16,6 +16,10 @@ menu:
 # Example 1: Data Import in MeVisLab
 MeVisLab provides several pre-defined modules to import data for processing in your networks.
 
+{{<alert class="info" caption="Extra Infos">}}
+The easiest way to load data in MeVisLab is to drag and drop the file into the MeVisLab workspace. MeVisLab will try to find a module which is able to load your file automatically.
+{{</alert>}}
+
 This chapter explains the basic data formats and modules to use in your network:
```

Comment:
These chapters explain the data formats and modules suitable for this specific use case/data formats and modules related to this use case/used in the example/data formats and modules you might want to look into.